easybuild/easyconfigs/g/GSL/GSL-1.15-goolf-1.7.20.eb

```diff
@@ -0,0 +1,18 @@
+easyblock = 'ConfigureMake'
+
+name = 'GSL'
+version = '1.15'
+
+homepage = 'http://www.gnu.org/software/gsl/'
+description = """The GNU Scientific Library (GSL) is a numerical library for C and C++ programmers.
+The library provides a wide range of mathematical routines such as random number generators, special functions and least-squares fitting."""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+toolchainopts = {'opt': True, 'optarch': True, 'unroll': True, 'pic': True}
+
+source_urls = ['http://ftpmirror.gnu.org/gsl/']
```

Comment:
personally I prefer this url. It's easier to find out the url in case you want to check